uint8 update_state
       
uint8 chassis_led_state_all_off = 0
uint8 chassis_led_state_red_on  = 1
uint8 chassis_led_state_bule_on = 2
uint8 chassis_led_state_green_on = 3
uint8 chassis_led_state_red_flash = 4
uint8 chassis_led_state_green_flash = 5
uint8 chassis_led_state_blue_flash  = 6